Kentico/custom-element-samples
# DCN-111 - Optimizely audiences selector

### Motivation

https://kentico.atlassian.net/browse/DCN-111

### How to test

See readme on how to use it, test various editing with the selector. Configuration for sample project available in related article (through JIRA task)
